spring-cloud-skipper-shell-commands/src/main/resources/usage.txt

```diff
@@ -0,0 +1,9 @@
+Skipper Options:
+  --skipper.uri=<uri>                              Address of the Skipper Server [default: http://localhost:9494].
+  --skipper.username=<USER>                        Username of the Skipper Server [no default].
+  --skipper.password=<PASSWORD>                    Password of the Skipper Server [no default].
+  --dataflow.credentials-provider-command=<COMMAND> Executes an external command which must return an OAuth Access Token [no default].
```

Comment:
Changed prefix to be consistent w/ other Spring Cloud projects.  Now have

```
@ConfigurationProperties("spring.cloud.skipper.client")
public class SkipperClientConfigurationProperties {
```

and
```
@ConfigurationProperties("spring.cloud.skipper.server")
public class SkipperServerProperties {
```